cli/types/index.d.ts

```diff
@@ -2170,11 +2170,19 @@ declare namespace Cypress {
     height: number
   }
 
+  type Padding =
+    | number
+    | [number]
+    | [number, number]
+    | [number, number, number]
+    | [number, number, number, number];
+
   interface ScreenshotOptions {
     blackout: string[]
     capture: 'runner' | 'viewport' | 'fullPage'
     clip: Dimensions
     disableTimersAndAnimations: boolean
+    padding: Padding

```

Comment:
This looks good to me. I was surprised for a moment that nothing was optional, but I see that this is used in combination with `Partial<T>` elsewhere in the file.